<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>客户端主界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>我的信息</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid #014F84
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imges.qrc">
     <normaloff>:/img/touxiang.png</normaloff>:/img/touxiang.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_tosend">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,1">
    <item>
     <widget class="QTextEdit" name="textEdit_content">
      <property name="styleSheet">
       <string notr="true">QTextEdit
{

border-radius:0px;
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>在线成员</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">QListWidget{
border-radius:0px;
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>74</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_skin">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius:0px;
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255,100);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imges.qrc">
        <normaloff>:/img/skin.png</normaloff>:/img/skin.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/new/prefix1/imgs//min.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255,100);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imges.qrc">
        <normaloff>:/img/min.png</normaloff>:/img/min.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/new/prefix1/imgs/quit.png);
background-repeat:no-repeat;
border:none;
background-position:center center;

}
QPushButton:hover{

background-color: rgba(230,0,0,0.5);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imges.qrc">
        <normaloff>:/img/quit.png</normaloff>:/img/quit.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imges.qrc"/>
 </resources>
 <connections/>
</ui>
